schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,231 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-11-01/Microsoft.Network.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Network",
+  "description": "Microsoft Network Resource Types",
+  "resourceDefinitions": {
+    "trafficManagerProfiles": {
+      "type": "object",
+      "description" : "An Azure Traffic Manager profile",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2015-11-01"
+          ]
+        },
+        "type": {
+          "enum": [
+            "Microsoft.Network/trafficManagerProfiles"
+          ]
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "profileStatus": {
+              "anyOf": [
```

Comment:
Wenjun, I had to change all of the "oneOf" clauses to "anyOf" because the positive tests (see the test file) were returning this error: "Data is valid against more than one schema from \"oneOf\": indices 0 and 1".